poc/prg.sage

```diff
@@ -110,6 +110,17 @@ if __name__ == "__main__":
 
     cls = PrgAes128
     test_prg(cls, Field128, 23)
+
+    # Test expand_into_vec, exercise rejection sampling
```

Comment:
Would you mind copy-pasting this comment here? https://github.com/divviup/libprio-rs/pull/222/files#diff-ee53018b069fa1ccd59e63911e683c27b728af0963b465436b100019b691808eR192-R193